localisation/english/xvcv_mdlc_l_english.yml

```diff
@@ -803,7 +803,7 @@
  xvcv_mdlc_common_district_mining_desc: "This area is designated to disassemble various facilities and machines within. They have considerable potentials, but all meaningless as we don't know how to use them."
  xvcv_mdlc_common_district_mining_plural: "Installation Deconstruction Sites"
  xvcv_mdlc_common_district_cooling: "Cooling System"
- xvcv_mdlc_common_district_cooling_desc: "This mega cooling facility is essential for preventing overheating and a breakdown issue of world-machines. One facility can support approximately three (3) production districts."
+ xvcv_mdlc_common_district_cooling_desc: "This mega cooling facility is essential for preventing overheating and breakdown of world-machines. Every 10% of district bonus supports one production district."
```

Comment:
I would tweak it to be 

    This mega cooling facility is essential for preventing overheating and breakdown of specialized world-machines. One facility can support approximately three (3) production districts.

The "every 10%" part confuses me a bit, I prefer the original wording.
On further reflection I suggest "specialized" since it's not part of the basic Auto-Machine World